packages/plugin-http-client/src/rest/main.ts

```diff
@@ -0,0 +1,7 @@
+export { AbstractResource } from './AbstractResource';
+export { BaseEntity } from './BaseEntity';
+export { EntityProcessor } from './processor/EntityProcessor';
+export { EntityMapper } from './mapper/EntityMapper';
+export { EntityListMapper } from './mapper/EntityListMapper';
+export { DefaultToArray } from './mapper/DefaultToArray';
+export { BaseMapper } from './mapper/BaseMapper';
```

Comment:
Múžeme přidat i exporty různých typů co se tam používají? `EntityConstructor`, `Mapper` apod ať se to potom dá použít v rámci aplikace

Reply:
Snad jsem popřidával vše. Snad muže byt i interface pod export type.